### PR TITLE
Update/2.2 fixed pep8 warnings

### PR DESCRIPTION
**Pull request by ulfox**
Warnings printed from: flake8 --statistics ./lenses_python/ setup.py

### CODE REVIEW

File: lenses_python/ConnectorHandler.py [hunk 1]
```diff
@@ -11,7 +12,7 @@ def __init__(self, url, username, password, token):
         self.token = token
         self.url_extend = CONNECTOR_END_POINT  # /api/proxy-connect/
         self.default_headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
-                            'x-kafka-lenses-token': self.token}
+                                'x-kafka-lenses-token': self.token}
```
Comment (spirosoik) [inline lenses_python/ConnectorHandler.py]: better to have this kind of format is more readable and you will avoid also the extra space you have added:

```python
{
   'Content-Type': 'application/json', 'Accept': 'application/json',
   'x-kafka-lenses-token': self.token
}
```

File: lenses_python/KerberosTicket.py [hunk 2]
```diff
@@ -11,7 +12,8 @@ def __init__(self, service):
         kerberos.authGSSClientStep(krb_context, "")
         self._krb_context = krb_context
         # Create the value of authentication header
-        self.auth_header = ("Negotiate " +kerberos.authGSSClientResponse(krb_context))
+        self.auth_header = ("Negotiate " + kerberos.authGSSClientResponse(krb_context))
```
Comment (spirosoik) [inline lenses_python/KerberosTicket.py]: use string interpolation

File: lenses_python/ProcessorHandler.py [hunk 3]
```diff
@@ -45,12 +45,11 @@ def CreateProcessor(self, name, sql, runners, clusterName, namespace, pipeline):
                 clusterName=clusterName
             )
 
-
         headers = {'Content-Type': 'application/json', 'Accept': 'text/plain',
-                    'x-kafka-lenses-token': self.token}
+                   'x-kafka-lenses-token': self.token}
```
Comment (spirosoik) [inline lenses_python/ProcessorHandler.py]: same format as above.

File: lenses_python/ProcessorHandler.py [hunk 4]
```diff
@@ -45,12 +45,11 @@ def CreateProcessor(self, name, sql, runners, clusterName, namespace, pipeline):
...
         response = post(url, headers=headers, json=params)
         if response.status_code != 200:
-            raise Exception("Http status code {}.{}".format(response.status_code,response.text))
+            raise Exception("Http status code {}.{}".format(response.status_code, response.text))
```
Comment (spirosoik) [inline lenses_python/ProcessorHandler.py]: use string interpolation.

File: lenses_python/SchemasHandler.py [hunk 5]
```diff
@@ -25,7 +26,7 @@ def ListAllSubjects(self):
         url = self.url+self.schemas_end_point
         response = get(url, headers=self.default_headers)
         if response.status_code != 200:
-            raise Exception("Http status code {}.{}".format(response.status_code,response.text))
+            raise Exception("Http status code {}.{}".format(response.status_code, response.text))
```
Comment (spirosoik) [inline lenses_python/SchemasHandler.py]: same

File: lenses_python/SchemasHandler.py [hunk 6]
```diff
@@ -39,7 +40,7 @@ def ListVersionsSubj(self, subject):
         url = self.url + self.schemas_end_point + '/' + subject + "/versions"
         response = get(url, headers=self.default_headers)
         if response.status_code != 200:
-            raise Exception("Http status code {}.{}".format(response.status_code,response.text))
+            raise Exception("Http status code {}.{}".format(response.status_code, response.text))
```
Comment (spirosoik) [inline lenses_python/SchemasHandler.py]: same and in all the cases like this.

File: lenses_python/SchemasHandler.py [hunk 7]
```diff
@@ -77,7 +78,7 @@ def GetSchemaByVer(self, subject, verid):
         :param verid:
         :return:
         """
-        url = self.url + self.schemas_end_point + '/' +subject + "/versions/"+verid
+        url = self.url + self.schemas_end_point + '/' + subject + "/versions/" + verid
```
Comment (spirosoik) [inline lenses_python/SchemasHandler.py]: same here use string interpolation

File: lenses_python/SqlHandler.py [hunk 8]
```diff
@@ -100,7 +103,7 @@ def browsing_data(self, stats=0,  is_extract_pandas=False):
         if not is_extract_pandas:
             return {"records": data_list,
                     "stats": stats_list
-                   }
+                    }
```
Comment (spirosoik) [inline lenses_python/SqlHandler.py]: you have extra space. there is no need tor two line in general here. except if you exceed line limit by pep8

File: tests/test_sql_handler.py [hunk 9]
```diff
@@ -1,9 +1,48 @@
 class TestSqlHandler:
 
-    def test_SqlHandler(self, lenses_conn):
+    def test_SqlHandler_create_table(self, lenses_conn):
```
Comment (spirosoik) [inline tests/test_sql_handler.py]: let's change the method name with lowercase for all the words.

File: tests/test_sql_handler.py [hunk 10]
```diff
@@ -1,9 +1,48 @@
...
+
+        assert type(result_dt['records']) is list
+
+        query_st = (
+            "SHOW TABLES"
+        )
+
+        result_st = lenses_conn.SqlHandler(query_st)
+
+        _status=True
```
Comment (spirosoik) [inline tests/test_sql_handler.py]: there is no need for underscore.

Comment (ulfox) [reply tests/test_sql_handler.py]: Done

File: lenses_python/ProcessorHandler.py (hunk 3 above)
Comment (ulfox) [reply lenses_python/ProcessorHandler.py]: Moved the dictionary format to the what was suggested

File: lenses_python/KerberosTicket.py (hunk 2 above)
Comment (ulfox) [reply lenses_python/KerberosTicket.py]: We will move to string interpolation on a new branch, since this needs much work (changing almost all files :P)

**Pull request merged**